clj-refactor.el

```diff
@@ -3992,18 +4000,23 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-change-function-
       (pop-to-buffer cljr--change-signature-buffer))))
 
 ;;;###autoload
-(defun cljr--inject-jack-in-dependencies ()
+(defun cljr--inject-jack-in-dependencies (cider-jack-in &rest args)
   "Inject the REPL dependencies of clj-refactor at `cider-jack-in'.
 If injecting the dependencies is not preferred set `cljr-inject-dependencies-at-jack-in' to nil."
-  (when (and cljr-inject-dependencies-at-jack-in
-             (boundp 'cider-jack-in-lein-plugins)
-             (boundp 'cider-jack-in-nrepl-middlewares))
-    (add-to-list 'cider-jack-in-lein-plugins `("refactor-nrepl" ,(cljr--version t)))
-    (add-to-list 'cider-jack-in-nrepl-middlewares "refactor-nrepl.middleware/wrap-refactor")))
+  (if (and (not (string-empty-p (cljr--project-dir)))
+           cljr-inject-dependencies-at-jack-in)
```

Comment:
keeping the `boundp` checks on the cider vars make sense to make sure that the cider version installed is compatible with this feature... any particular reason you got rid of them?

Reply:
Oops, I was originally thinking we didn't need them since I switched from `add-to-list` to `let` + `cons`, but we still do since I use the original value. Fixing that now.